Dockerfile.konflux

```
@@ -14,25 +14,26 @@ COPY cmd/ cmd/
-    CGO_ENABLED=${CGO_ENABLED} GOOS=${TARGETOS} GOARCH=${TARGETARCH} \
-    GOEXPERIMENT=strictfipsruntime \
-    go build -a -o bin/manager ./cmd/
-
 # Obtain the batch-gateway Helm chart into /tmp/batch-gateway-chart.
 COPY Makefile ./
-RUN if [ "${BUILD_TYPE}" = "CI" ]; then \
-        # hermetic: chart provided by the Konflux prefetch step (path set by the downstream prefetch config).
-        cp -r /cachi2/prefetched-charts/batch-gateway /tmp/batch-gateway-chart; \
-    else \
+
+COPY Dockerfile prefetched-charts/batch-gateway/ /tmp/batch-gateway-chart
```

Hadolint (2.14.0)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)